### PR TITLE
Minor changes for Hive view support

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

**Comment by sopel39** [comment]
@electrum thanks. you also need to enable view execution in product tests

File: presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConfig.java [hunk 1]
```diff
@@ -597,7 +597,7 @@ public boolean isHiveViewsEnabled()
         return hiveViewsEnabled;
     }
 
-    @Config("hive.views-execution.enabled")
+    @Config("hive.view-execution.enabled")
```
Comment (findepi) [inline presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConfig.java]: or maybe even `hive.hive-view...` ? to indicate it's _hive_ connector's property for _hive views_

Comment (electrum) [reply presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConfig.java]: Maybe we should call this `translate-hive-views`?

File: presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConfig.java [hunk 2]
```diff
@@ -592,16 +593,17 @@ public HiveConfig setFileStatusCacheTables(String fileStatusCacheTables)
...
-        return hiveViewsEnabled;
+        return translateHiveViews;
     }
 
-    @Config("hive.views-execution.enabled")
-    @ConfigDescription("Allow execution of Hive views")
-    public HiveConfig setHiveViewsEnabled(boolean hiveViewsEnabled)
+    @LegacyConfig("hive.views-execution.enabled")
+    @Config("hive.translate-hive-views")
+    @ConfigDescription("Allow translation of Hive views into Presto views")
```
Comment (findepi) [inline presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConfig.java]: One more thing -- should this (description / toggle name) say it's experimental?
We don't have proper translation layer yet.

File: presto-hive/src/main/java/io/prestosql/plugin/hive/util/HiveUtil.java [hunk 3]
```diff
@@ -670,16 +672,18 @@ public static ConnectorViewDefinition decodeViewData(String data)
...
                 Optional.ofNullable(view.getDatabaseName()),
                 view.getDataColumns().stream()
-                        .map(column -> new ConnectorViewDefinition.ViewColumn(column.getName(), TypeId.of(column.getType().getTypeSignature().toString())))
+                        .map(column -> new ViewColumn(column.getName(), TypeId.of(column.getType().getTypeSignature().toString())))
                         .collect(toImmutableList()),
-                Optional.ofNullable(view.getOwner()),
-                Optional.empty(), // no comment
-                false); // don't run as invoker);
+                Optional.ofNullable(view.getParameters().get(TABLE_COMMENT)),
+                Optional.of(view.getOwner()),
```
Comment (findepi) [inline presto-hive/src/main/java/io/prestosql/plugin/hive/util/HiveUtil.java]: @sopel39 comment/owner were reversed. Should  we have a test? https://github.com/prestosql/presto/issues/2753

Comment (sopel39) [reply presto-hive/src/main/java/io/prestosql/plugin/hive/util/HiveUtil.java]: good idea

File: presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConfig.java (hunk 2 above)
Comment (electrum) [reply presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConfig.java]: Good idea. Added.

**Pull request merged**